cmd/cluster/log-exporter/query_log_exporter.go

```diff
@@ -31,7 +31,7 @@ import (
 var ClusterName string
 
 var DbQueryLoggingCmd = &cobra.Command{
-	Use:   "db-query-logging",
+	Use:   "db-query-log-exporter",
```

Comment:
Is this released to the public already? If so, we can't really change it. If it is not released, then we can go ahead. Please let me know.

Reply:
DB Query Log feature is GA, CLI for it is being developed as part of this PR.